mtp_noms_ops/apps/security/tests/test_views/test_checks.py

```diff
@@ -401,31 +414,18 @@ def test_displays_count_of_credits_needing_attention(self, mock_get_need_attenti
             self.assertIn('2 credits need attention', content)
             self.assertIn('This credit needs attention today!', content)
 
-    def test_calculation_of_date_before_which_checks_need_attention(self):
```

Comment:
this is now covered by other existing tests because mock response in `mock_need_attention_count` requires the `started_at__lt` parameter to be correct